src/core/mtu_discovery.c

```diff
@@ -81,6 +81,15 @@ QuicGetNextProbeSize(
     // depends on that behavior.
     //
 
+    //
+    // Jump automatically to 1280 to return algorithm to ideal case. 1280 should
+    // be supported in most scenarios. With minimum being 1248, this will always
+    // be less then a full increment.
+    //
+    if (Path->Mtu < 1280) {
```

Comment:
Doesn't this prevent the app from ever setting the max to less than 1280?

Reply:
Oh, yeah good point. Let me fix that.